src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -359,26 +373,15 @@ private MinigameTask cloneBugfinder(
             return new MinigameTask(Minigame.BUGFINDER, minigameTask.getDescription(), null, minigameTask.getIndex());
         } else {
             try {
-                BugfinderConfiguration config = bugfinderClient.getConfiguration(
-                    accessToken,
-                    minigameTask.getConfigurationId()
-                );
-                config.setId(null);
-                config
-                    .getCodes()
-                    .forEach(bugfinderCode -> {
-                        bugfinderCode.setId(null);
-                        bugfinderCode.getWords().forEach(bugfinderWord -> bugfinderWord.setId(null));
-                    });
-                config = bugfinderClient.postConfiguration(accessToken, config);
+                final UUID cloneId = bugfinderClient.postClone(accessToken, minigameTask.getConfigurationId());
                 return new MinigameTask(
                     Minigame.BUGFINDER,
                     minigameTask.getDescription(),
-                    config.getId(),
+                    cloneId,
                     minigameTask.getIndex()
                 );
             } catch (final FeignException e) {
-                log.debug(CLONE_ERROR_MESSAGE, e);
+                log.info(CLONE_ERROR_MESSAGE, e);
```

Comment:
Erm… I think we need to have a discussion about appropriate log levels…
```suggestion
                log.error(CLONE_ERROR_MESSAGE, e);
```